### PR TITLE
docs: add initialization of LogEntry instance in the v2 example

### LINKED ISSUE #44: getting started sample shows how to iterate over empty logs list

### PR DESCRIPTION
**Pull request by ymotongpoo**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-logging/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [X] Appropriate docs were updated (if necessary)

Fixes #44

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -84,7 +84,9 @@ Using the API
     from google.cloud import logging_v2
 
     client = logging_v2.LoggingServiceV2Client()
-    entries = []
```
Comment (ymotongpoo) [inline README.rst:87]: Though this PR only changed the issue that the list `entries` is empty, I think we need to add how to set `resources` and `log_name`. What do you think?

Comment (simonz130) [reply README.rst:87]: Yes, that would be useful to have too.

File: README.rst [hunk 2]
```diff
@@ -84,7 +84,9 @@ Using the API
...
+    e = logging_v2.types.LogEntry(
+        text_payload="text")
```
Comment (simonz130) [inline README.rst:88]: please update "Text" to something more meaningful, like "this is a log statement"

File: README.rst (hunk 1 above)
Comment (ymotongpoo) [reply README.rst:87]: Thanks, I added the note.

File: README.rst (hunk 2 above)
Comment (ymotongpoo) [reply README.rst]: I used the exact phrase :)

**Comment by ymotongpoo** [comment]
@simonz130 Thank you Simon for the check. I updated the pull request.

**Pull request merged**